src/test/java/uk/nhs/hee/trainee/details/config/MongoIndexConfigurationTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright 2021 Crown Copyright (Health Education England)
```

Comment:
Not something important, just the year not updated

Reply:
I left this as the original year because it's essentially a renamed `MongoConfiguration` to `MongoIndexConfiguration` and created a new `MongoConfiguration`... it's a bit of a semantical argument where it should go, but I left the original copyright with the  bulk of the original code.